lib/novu/http.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Novu.Http do
+  @moduledoc """
+  Helper functions for interacting with Req.
+  """
+  require Logger
+
+  @api_key Application.compile_env(:novu, :api_key)
```

Comment:
With these using `compile_env` will they be able to get configuration from `runtime.exs` in the root application?

Reply:
That's a good call out. I think we should be fine but I moved these to functions anyways for another reason: I could see us wanting to having it fall through from ENV -> App Config or vice versa and that'd be easier to accomplish in a function.